Dockerfile-linux.template

```
@@ -307,7 +307,10 @@ RUN set -eux; \
 		"pip==$PYTHON_PIP_VERSION" \
 {{ if .setuptools then ( -}}
 		"setuptools==$PYTHON_SETUPTOOLS_VERSION" \
-{{ ) else "" end -}}
+{{ ) else ( -}}
+		--no-setup-tools \
```

```suggestion
		--no-setuptools \
```

https://github.com/pypa/get-pip/blob/ac9253767f0be4cdef620a3ca07187a8cabc66a5/README.md#get-pippy-options :eyes: